.github/workflows/continuous-integration.yml

```diff
@@ -183,18 +182,6 @@ jobs:
         run: ./scripts/ci_steps/install_native_linux_dependencies.sh
       - name: Tools Versions
         run: ./scripts/ci_steps/log_tool_versions.sh
-      # Configure fake credentials using profiles
-      - name: (Credentials 1/3) Create AWS config directory
-        run: mkdir -p ~/.aws
-      - name: (Credentials 2/3) Create AWS config file
-        run: |
-          echo "[default]" > ~/.aws/config
-          echo "region=us-west-2" >> ~/.aws/config
-      - name: (Credentials 3/3) Create AWS credentials file
-        run: |
-          echo "[default]" > ~/.aws/credentials
-          echo "aws_access_key_id = test-key-id" >> ~/.aws/credentials
-          echo "aws_secret_access_key = test-secret-access-key" >> ~/.aws/credentials
```

Comment:
The code above is deleted since credentials are now provided statically at protocol test client creation.